monitor/pom.xml

```diff
@@ -132,6 +132,12 @@
 					</exclusion>
 				</exclusions>
 			</dependency>
+			<!-- spring-security-cas imports version 3.1, while hadoop-common imports version 3.2.1. -->
+			<dependency>
+				<groupId>commons-collections</groupId>
+				<artifactId>commons-collections</artifactId>
+				<version>3.2.1</version>
```

Comment:
No yet. This new addition is in the <dependencyManagement> part of monitor pom.xml